scripts/vehicles.py

```diff
@@ -4,17 +4,18 @@
 import propertiesReader
 import requests
 
-authToken = authTokenGenerator.getAuthToken()
-vehiclesUrl = propertiesReader.getVehiclesUrl()
-
-headers = {
-    'User-Agent':propertiesReader.getUserAgent(),
-    'Content-Type': "application/json",
-    'Authorization': "Bearer " + authToken
-}
 
 def getVehicles():
-	res = requests.request("GET", vehiclesUrl, headers=headers)
-	resJson = res.json()
-	response = resJson['response']
-	print(response)
\ No newline at end of file
+    authToken = authTokenGenerator.getAuthToken()
+
+    headers = {
+        'User-Agent':propertiesReader.getUserAgent(),
+        'Content-Type': "application/json",
+        'Authorization': "Bearer " + authToken
+    }
+
+    vehiclesUrl = propertiesReader.getVehiclesUrl()
+    response = requests.request("GET", vehiclesUrl, headers=headers)
+    json = response.json()
+    resJson =  json['response'][0]
```

Comment:
good for v1 but I am already facing an issue where I'm not getting my vehicle's status because it's not first in the list. Will file an enhancement for this case.

Reply:
Yeah we can enhance that for sure